### ISSUE TITLE
Implement Combat transformer

### ISSUE THREAD

**Issue by richford** [richford/AFQ-Insight#62]
From @mnarayan:

> I did a tutorial on dealing with site effects a few years ago at OHBM, I can send my slides around. A few papers have shown that effectively standardizing DTI features to make sure they look similar across all sites (using a combat) was quite useful. https://github.com/neuroquant/neurocombat. It might be useful to use this as part of the model training (i.e. learning the combat standardization model on the training set and applying it on the test set before calling predict).

We should implement combat as an sklearn feature transformer and allow users the option of inserting it into the pipeline.

**Comment by richford** [comment]
Closing since [neurocombat_sklearn](https://github.com/Warvito/neurocombat_sklearn) does exactly this.
